src/Webapi/Dom/Webapi__Dom__RectList.res

```diff
@@ -0,0 +1,9 @@
+type t = {
+  length: int
+}
+
+@val external toArray: t => array<Dom.domRect> = "Array.prototype.slice.call"
+
+@get external length: t => int = ""
+
+@send @return(nullable) external item: (t, int) => option<Dom.domRect> = ""
```

Comment:
Should this be = "item" to remove that warning about property names being non explicit.